tests/semanticdb/expect/InventedNames.scala

```diff
@@ -32,7 +32,7 @@ given [T]: Z[T] with
 
 val a = intValue
 val b = given_String
-//val c = given_Double
+val c = given_Double
```

Comment:
This will fail since commit https://github.com/scala/scala3/commit/5a6ca9d23559e9a4c8e06aace3649cdb3c1eaef4 also adapted the `.expect` and `.expect.scala` files to satisfy the changes in the tests